server/common.ts

```diff
@@ -329,14 +330,14 @@ export function authenticateWithReject(request: express.Request, response: expre
 // For directly user facing endpoints
 export function authenticateWithRedirect(request: express.Request, response: express.Response, next: express.NextFunction) {
 	if (!request.isAuthenticated()) {
+		trackEvent("visit", request.ip);
```

Comment:
My thought process here was that people who don't already have accounts will probably be trying to access `hack.gt` or `hack.gt/apply`  and their visits would get logged at this point. However, tracking all visits would be a good thing to do. I'll take another shot at this tomorrow; thanks for looking over this!